spec/support/email_helpers.rb

```diff
@@ -0,0 +1,53 @@
+module EmailHelpers
```

Comment:
am i right in thinking we're not using this module any more?

Reply:
Apparently so.  I've removed it now.